exercises/modal.js

```diff
@@ -32,7 +37,9 @@ class Modal{
 				//this.FUNCTIONNAME.bind( this )
 			//otherwise it won't work correctly
 	init(){
-
+		this.hide();
+		this.modalShadow.click(this.onClose.bind(this));
```

Comment:
I usually prefer to bind this stuff in the constructor so I don't have the possibility of unbound functions being used.  Also keeps fewer bound functions for removing handlers and memory leakage, but that would be very minor.